.github/workflows/bash.yml

```diff
@@ -6,18 +6,62 @@ on:
     types: [synchronize, opened, reopened, ready_for_review]
     paths:
     - '**.bash'
```

Comment:
Maybe add *.sh files too

Reply:
Bad idea in bash since we need to parse `--shell=sh` to shellcheck

Adapting the logic in bash is also not good idea since it would be processing unwanted files.

i.e if there are 2 files:
- foo.bash
- foo.sh

Then that would had to process both of these files instead if checking only one per extension. (generally faster)

Reply:
OK